sites/public/src/components/listings/Pagination.tsx

```diff
@@ -67,31 +72,48 @@ export function Pagination(props: PaginationProps) {
   // className={styles["pagination"]}
   // size={AppearanceSizeType.small}
 
+  if (props.lastPage === 1) return <></>
```

Comment:
Hides pagination if there is only one page.

Prod:
<img width="600" alt="Screenshot 2024-11-12 at 8 21 21 PM" src="https://github.com/user-attachments/assets/88610c49-d012-4a1d-84cc-9b7a61883e7e">